docs/ABOUT.md

```diff
@@ -1,11 +1,9 @@
-Julia is an open-source high-level, high-performance dynamic programming language designed for technical and scientific computing while also being effective for general-purpose tasks. It is convenient to use for daily work but also runs fast enough to be deployed for high-performance applications.
-
-Interesting features include:
+Some interesting features of the language include:
 - Large parts of [Julia's base library](https://github.com/julialang/julia) are written in Julia itself. Understanding and contributing to the Julia core does not require knowledge of another language.
 - Easy to use interfaces to call libraries written in other languages, such as [C, Fortran](http://docs.julialang.org/en/stable/manual/calling-c-and-fortran-code/) and [Python](https://github.com/JuliaPy/PyCall.jl), directly.
 - [Multiple dispatch](http://docs.julialang.org/en/stable/manual/methods/#man-methods)
 - A dynamic, nominative and parametric [type system](http://docs.julialang.org/en/stable/manual/types/).
-- Homoiconicity: Julia code can be represented in Julia itself, making it a good language to learn about [metaprogramming](http://docs.julialang.org/en/stable/manual/metaprogramming/).
+- Homoiconicity: Julia code can be represented in Julia itself, which allows for powerful [metaprogramming](http://docs.julialang.org/en/stable/manual/metaprogramming/) uses.
 
 The first public release was in 2012. You can find out more about the motivation behind it in the blog post ["Why We Created Julia"](http://julialang.org/blog/2012/02/why-we-created-julia) by the core developers.
```

Comment:
That is a fantastic blog post.

I'd love it if the opening of the ABOUT could lead with the sentiments from the blog post.

Maybe something like:

> The creators of Julia want to eat their cake and have it too. As they describe in their blog post [Why We Created Julia](http://julialang.org/blog/2012/02/why-we-created-julia) they want the speed of C, the dynamism of Ruby, the familiar mathematical notation of Matlab. They want it to be their favorite things from their favorite languages. String processing like Perl. Glue like the shell. Powerful but not impenetrably complex. Country _and_ western.

In short, they want _[s]omething that is dirt simple to learn, yet keeps the most serious hackers happy_.

I also think that if we could rework the bullet points to be a couple of paragraphs of prose, that would read more smoothly.